src/services/organisation_service.py

```
@@ -638,24 +724,26 @@ async def fetch_cabinet_flow(self, president_id: str, dates: Sequence[str], max_
                 for (source, target), department_ids in links_departments.items()
             ]
-            
-            unique_ids = list({node['id'] for node in nodes})
+
+            unique_ids = list({node["id"] for node in nodes})
             tasks_for_getting_names = [
-                self.opengin_service.get_entities(
-                    entity= Entity(id=id)
-                )
+                self.opengin_service.get_entities(entity=Entity(id=id))
                 for id in unique_ids
```

Ruff (0.16.0)

[error] Variable `id` is shadowing a Python builtin

(A001)